traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go

```diff
@@ -274,6 +274,18 @@ func GetDSNameFromID(tx *sql.Tx, id int) (tc.DeliveryServiceName, bool, error) {
 	return name, true, nil
 }
 
+// GetDSCDNIdFromID loads the DeliveryService's cdn ID from the database, from the delivery service ID. Returns whether the delivery service was found, and any error.
+func GetDSCDNIdFromID(tx *sql.Tx, id int) (int, bool, error) {
+	var cdnID int
```

Comment:
`cdnID` is a good name because we know that it refers to the ID of a CDN. `id` is less clear, the variable name should mention what kind of ID it is